src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -1610,6 +1614,8 @@ def load_arguments(self, _):
             is_preview=True,
             help="delete an AKS nodepool by ignoring PodDisruptionBudget setting",
         )
+        c.argument("if_match")
+        c.argument("if_none_match")
```

Comment:
From API sepc, nodepool delete operation does not support request parameter `if_none_match`
https://github.com/Azure/azure-rest-api-specs/pull/28632

Reply:
Removed in latest commit